PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -22,9 +22,9 @@ public static void Main(string[] args)
             Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
         }
 
-        public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
+        public static string GetPercentCorrect(double numberCorrectAnswers, double numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((numberCorrectAnswers / numberOfQuestions) * 100.0) + "%";
```

Comment:
Since both numberCorrectAnswers and numberOfQuestions are stored as ints, the method parameters here should reflect that being the case. It's also not necessary to change them to doubles.

```suggestion
           double percentage = 100.0 * numberCorrectAnswers / numberOfQuestions;
           return Math.Round(percentage, 2) + "%";
```

Instead, something like this will prevent integer division from occuring. 
You could just explicitly cast to a double here, but even that is not necessary. 
Multiplication and division are of equal weight within the order of operations, so the multiplication will occur first. 
Once the multiplication occurs, the numerator becomes a double and forces the denominator to be promoted to a double. Thus, resolving the integer division problem.